tests/metrics/network/latency_kubernetes/latency-network.sh

```diff
@@ -18,6 +18,14 @@ function remove_tmp_file() {
 
 trap remove_tmp_file EXIT
 
+function latency_cleanup() {
+	info "Latency test cleanup"
+	kubectl delete -f latency-server.yaml
+	kubectl delete -f latency-client.yaml
+}
+
+trap latency_cleanup EXIT
```

Comment:
If the script ends before the server and clients pods are created, call to `latency_cleanup` could report error.
I can think of two options, :
- You can make that the lines that deletes the pods return always true
- or move the `trap` line just after the creation of the server and client pods.

Reply:
thanks @dborquez for the feedback changes applied